packages/eds-core-react/src/components/Button/tokens/button.ts

```diff
@@ -98,7 +105,11 @@ export const button: ButtonToken = {
   },
   modes: {
     compact: {
-      height: compactButtonHeight,
+      minHeight: '1.7142em', // 24px
+      spacings: {
+        top: 'calc(0.2857em - 1px)', // calc(4px - 1px) - 4/14
+        bottom: 'calc(0.2857em - 1px)',
+      },
       clickbound: {
```

Comment:
…and perhaps this too. What good are clickbounds when compact mode breaks with UU in the first place?